packages/docusaurus/src/commands/swizzle/actions.ts

```diff
@@ -125,7 +125,10 @@ export async function wrap({
 import ${componentName} from '@theme-${importType}/${themeComponentName}';
 import type ${componentName}Type from '@theme/${themeComponentName}';
 
-type Props = ComponentProps<typeof ${componentName}Type>;
+type RawProps = ComponentProps<typeof ${componentName}Type>;
+// ComponentProps is a little buggy, quick fix.
+// https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/60766
+type Props = unknown extends RawProps ? {} : RawProps;
```

Comment:
Ideally revert before merge—depends on if DT is open to fixing this.

Reply:
cf Discord

We'd rather export our own `ComponentProps` or `WrapperProps` type instead of inlining these more complex types

https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/60766